### PR TITLE
 UIORG-142: Refactor location locations functionality to get rid of `EntryManager` usage

### PR DESCRIPTION
**Pull request by VictorSoroka1**
Additionally includes changes for [UITEN-18](https://issues.folio.org/browse/UITEN-18).

### CODE REVIEW

File: src/settings/LocationLocations/LocationManager.js [hunk 1]
```diff
@@ -374,47 +485,203 @@ class LocationManager extends React.Component {
...
+    return (
+      <Paneset
+        defaultWidth="fill"
+        data-test-entry-manager
+      >
+        <Pane
+          defaultWidth="fill"
+          paneTitle={label}
+        >
+          <SearchAndSortQuery>
```
Comment (mkuklis) [inline src/settings/LocationLocations/LocationManager.js]: @VictorSoroka1 very nice to see `<SearchAndSortQuery>` being used here!

Comment (VictorSoroka1) [reply src/settings/LocationLocations/LocationManager.js]: Thank you @mkuklis, it really fit there)

File: src/settings/LocationLocations/LocationManager.js [hunk 2]
```diff
@@ -276,27 +370,31 @@ class LocationManager extends React.Component {
...
+    const {
+      resources,
+      location,
+      intl: { formatMessage },
+    } = this.props;
+    const {
+      institutionId,
+      campusId,
+      libraryId,
+    } = this.state;
```
Comment (YevheniiMaltsev) [inline src/settings/LocationLocations/LocationManager.js]: It looks like a blank line should be added after this line for code consistency

Comment (VictorSoroka1) [reply src/settings/LocationLocations/LocationManager.js]: Agree. Done.

**Pull request merged**